src/mame/drivers/gigatron.cpp

```diff
@@ -87,7 +114,7 @@ void gigatron_state::gigatron(machine_config &config)
 	screen.set_screen_update(FUNC(gigatron_state::screen_update));
 
 	/* sound hardware */
-	//SPEAKER(config, "mono").front_center();
+	SPEAKER(config, "mono").front_center();
```

Comment:
This is still a premature addition that will fail due to a lack of input sound channels.

Reply:
Fixed.